project/Dependencies.scala

```diff
@@ -37,19 +37,21 @@ object Dependencies {
   val scalaTestVersion = "3.1.4"
   val scalaCheckVersion = "1.15.1"
 
+    def getScalaVersion() = {
+    // don't allow full override to keep compatible with the version of silencer
```

Comment:
I guess this comment doesn't apply anymore as we don't use the silencer anymore?

Reply:
Good catch! This is an interesting subject, in the Scala 3 PR I'm heavily relying on this method, even if it looks like is not needed anymore, can I just remove the comment for now?